From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Tue, 23 Aug 2016 10:44:56 +0200
Subject: [PATCH] MAINTAINERS: document ASoC header files

include/sound/simple_card_utils.h is handled by ASoC maintainers, as
stated in https://lkml.org/lkml/2016/8/22/307, and
include/sound/simple_card.h seems to be an ASoC file too. In the future
there will be more files named like these ones so introduce a pattern to
match them.

Signed-off-by: Nicolas Iooss <nicolas.iooss_linux@m4x.org>

Sent-upstream: https://lkml.org/lkml/2016/8/24/526
---
 MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 6781a3febd59..8f30a0bcd73e 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -11036,6 +11036,7 @@ S:	Supported
 F:	Documentation/devicetree/bindings/sound/
 F:	Documentation/sound/alsa/soc/
 F:	sound/soc/
+F:	include/sound/simple*card*
 F:	include/sound/soc*
 
 SOUND - DMAENGINE HELPERS
-- 
